### PR TITLE
New Keywords: break-at (foreach) / count (setvar)

### PR DESCRIPTION
**Pull request by oboecker**
This branch introduces two new keywords:
- break-at for "for" loops
-- this expects a parameter after the "break-at:" keyword
- count for "setvar" assignments
-- this expects a parameter after the "count:" keyword

For this purpose, a new Method has been introduced to obtain the Token for a given keyword. The logic is quite simple, but I am not sure how it can be tested.
Both keywords are realized using Traits, and rely on the token mechanism for validation and processing.

### CODE REVIEW

File: src/Mailcode/Traits/Commands/Validation/BreakAtTrait.php
Comment (Mistralys) [inline src/Mailcode/Traits/Commands/Validation/BreakAtTrait.php:1]: This has `use phpDocumentor\Descriptor\Interfaces\FunctionInterface;` which I suspect is a mistake - at least it should not be used here, as it is a dev-only library.

File: src/Mailcode/Traits/Commands/Validation/CountTrait.php
Comment (Mistralys) [inline src/Mailcode/Traits/Commands/Validation/CountTrait.php:1]: This has `use phpDocumentor\Descriptor\Interfaces\FunctionInterface;` as well, which should be removed.

File: src/Mailcode/Traits/Commands/Validation/NoTrackingTrait.php
Comment (Mistralys) [inline src/Mailcode/Traits/Commands/Validation/NoTrackingTrait.php:1]: There are some phpDocumentor use statements here as well.

File: src/Mailcode/Commands/CommonConstants.php
Comment (Mistralys) [inline src/Mailcode/Commands/CommonConstants.php:1]: This has whitespace changes only?

**Review by Mistralys — CHANGES_REQUESTED** [review]
I had a look at the files, beyond the phpDocumentor `use` statements that should be removed, it looks good!

File: src/Mailcode/Traits/Commands/Validation/NoTrackingTrait.php
Comment (oboecker) [reply src/Mailcode/Traits/Commands/Validation/NoTrackingTrait.php:1]: My guess is that this is the original source, as I C&P the other Traits from this one.

Comment (Mistralys) [reply src/Mailcode/Traits/Commands/Validation/NoTrackingTrait.php:1]: Ah, then it was most likely my bad :|

**Review by Mistralys — APPROVED** [review]

**Pull request merged**